pre-commit.sh

```diff
@@ -12,5 +12,5 @@ then
   echo "warning: VERIFY_SERVICE_PROVIDER_HOST not set, using localhost:50400 by default"
 fi
 
-npm test
-npm run acceptance-tests
+yarn test
+yarn run acceptance-tests-paas
```

Comment:
Is this not what pre-commit-paas is for?

Reply:
Didn't know that was there. I feel like pre-commit is a protected name at Verify and it should "just work"

Reply:
Maybe. But yeah, I'm not sure on whether we ever want to run the stuff not against paas (and if not we should probably just remove the not-against-paas scripts altogether and rename the paas ones). 

I can't merge this anyway, so up to someone who can.

(and see the readme - there's a -paas script for acceptance tests, pre-commit, and startup)